weave/trace_server/project_version/providers/memory_cache_provider.py

```diff
@@ -0,0 +1,69 @@
+"""In-memory LRU cache provider for project versions."""
+
+import logging
+from typing import Optional
+
+from cachetools import LRUCache
+
+from weave.trace_server.project_version.types import ProjectVersion
+
+logger = logging.getLogger(__name__)
+
+PER_REPLICA_CACHE_SIZE = 10_000
+
+
+class InMemoryCache:
```

Comment:
This a wrapper with essentially zero added business logic. Is this really necessary? Why not simply instantiate an `LRUCache` at the call site?